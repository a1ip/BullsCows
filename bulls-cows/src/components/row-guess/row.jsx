import React from 'react'
import style from './row.module.css'

export const Row = ({children}) => {
    return (
        <div className={style["row"]}>
            {children}
        </div>
    ) }