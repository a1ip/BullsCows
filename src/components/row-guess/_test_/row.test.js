import React from "react"
import renderer from 'react-test-renderer'

import {Row} from "../row"

describe('Row component', () => {

    const RowTest = renderer
        .create(<Row>
                    <p>Oh, sweet child of mine!</p>
                </Row>)

    it('renders correctly', () => {
        expect(RowTest.toJSON()).toMatchSnapshot()
    })

    it('renders children correctly', () => {
        expect(RowTest.root.children.length).toBe(1)
        expect(RowTest.root.findByType('p').children).toStrictEqual(['Oh, sweet child of mine!'])
    })
})