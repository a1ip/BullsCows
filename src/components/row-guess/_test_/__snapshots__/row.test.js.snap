// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row component renders correctly 1`] = `
<div
  className="row"
>
  <p>
    Oh, sweet child of mine!
  </p>
</div>
`;
